.changes/next-release/feature-AWSCommonRuntimeHTTPClient-dda0ac9.json

```diff
@@ -0,0 +1,6 @@
+{
+    "category": "__AWS Common Runtime HTTP Client__", 
```

Comment:
nit, is there any reason `__` is used as prefix and suffix? This may not render correctly with our existing changelog formatting. Same as other changelog entries

Reply:
removed and updated